string feature_name
FeatureModule feature_module
---
string[] possible_values
string[] available_values
Error error